apps/rewards/app/components/App/App.js

```diff
@@ -34,11 +77,28 @@ class App extends React.Component {
     balances: PropTypes.arrayOf(PropTypes.object),
   }
 
+  static defaultProps = {
+    network: {},
+  }
+
   state = {
     selected: 0,
     tabs: [ 'Overview', 'My Rewards' ],
   }
 
+  static childContextTypes = {
+    network: networkContextType,
+  }
+
+  getChildContext() {
```

Comment:
What does this function do? And where is it being called?

Reply:
That seems to be some historical things still hanging around. I've removed it.